pkgs/os-specific/linux/minimal-bootstrap/stage0-posix/default.nix

```diff
@@ -1,9 +1,13 @@
 { lib
 , newScope
+, make-minimal-bootstrap-sources
```

Comment:
This is in scope and not needed to be passed through

Reply:
52edaab39be9